convert2rhel/pkghandler.py

```diff
@@ -461,6 +468,25 @@ def replace_non_rhel_installed_kernel(version):
     utils.ask_to_continue()
 
     pkg = "kernel-%s" % version
+    ret_code = utils.run_subprocess('yum remove -y kmod-kvdo', print_output=False)
+    ret_code = utils.run_subprocess('yum remove -y libreport', print_output=False)
+    for pkgname in set(system_info.pkg_blacklist):
+        pkgname.replace("centos", "redhat")
+        utils.download_pkg(pkg=pkgname.replace("centos", "redhat"), dest=utils.tmp_dir,
+                           disablerepo=tool_opts.disablerepo, enablerepo=tool_opts.enablerepo)
+        ret_code = utils.run_subprocess('rpm -i --force --replacepkgs %s*' % os.path.join(utils.tmp_dir,
+                                                                                          pkgname.replace("centos",
+                                                                                                          "redhat")),
+                                        print_output=False)
+
+    for pkgname1 in set(system_info.pkg_blacklist):
+        pkgname1.replace("centos", "rhel")
+        utils.download_pkg(pkg=pkgname1.replace("centos", "rhel"), dest=utils.tmp_dir,
+                           disablerepo=tool_opts.disablerepo, enablerepo=tool_opts.enablerepo)
+        ret_code = utils.run_subprocess('rpm -i --force --replacepkgs %s*' % os.path.join(utils.tmp_dir,
+                                                                                          pkgname1.replace("centos",
+                                                                                                           "redhat")),
+                                        print_output=False)
```

Comment:
> Is it possible to replace rpm -i with utils.install_pkgs() as well? Actually, could we not simply call install directly instead of downloading and installing?

I believe install_pkgs don't have "replace" function like rpm.